list

```diff
@@ -153,6 +154,7 @@ j.mp
 jdem.cz
 kask.us
 kck.st
+kiwi.gg
```

Comment:
This one doesn't look like a normal url shortener?

Reply:
Not sure if it's because of the language, it doesn't look like a working URL shortener to me, are you be able to generate or access to any existing on the field shortened URL?

Reply:
No info about what it is (or the purpose of the project) on their website besides that it's in different language. Can't use the site to shorten any URL. Remove that from your side.